io-package.json

```diff
@@ -221,7 +221,8 @@
         },
         "dependencies": [
             {
-                "js-controller": ">=2.0.0"
+                "js-controller": ">=2.0.0",
+                "admin": ">=5.1.26"
```

Comment:
Admin needs to go into "common.globalDependencies" block!

Reply:
Whats the difference? And why there are two dependencie blocks?

Reply:
It is a multihost topic:

* dependencies: the dependency needs to be fullfilled on THIS host
* globalDependencies: The dependency needs to be fulfilled "in the system aka any host". the "lowest version of that adapter" is used then for check